mlflow/ml-package-versions.yml

```diff
@@ -526,9 +526,9 @@ paddle:
       "< 2.5.0": ["google-cloud-storage==2.14.0"]
     run: |
       pytest tests/paddle/test_paddle_model_export.py
-  autolog:
+  autologging:
```

Comment:
The key must be `autologging`

Reply:
Not a blocker, but can we update the pydantic models in `dev/set_matrix.py` to detect unrecognized fields?

Reply:
Let me do it in a follow-up🙂